sprites.py

```diff
@@ -260,9 +260,9 @@ def set_label(self, new_label, i=0):
             self.labels[i] = str(new_label)
         self.inval()
 
-    def set_margins(self, l=0, t=0, r=0, b=0):
+    def set_margins(self, w=0, t=0, r=0, b=0):
```

Comment:
"l" used to mean left edge.  "w" doesn't seem to mean left edge.  Same file is in several other activities, so you might check for how this was resolved elsewhere.